docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+services:
+  cvgenerator:
+    build:
+      context: .
+      dockerfile: dockerfile
+    image: unosquare/cvgenerator:latest
+    restart: always
+    environment:
+      - SERVER_PORT=8080
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://db:5432/postgres?user=postgres&password=password
+    ports:
+      - 8080:8080
+    depends_on:
+      - flyway
+  flyway:
+    image: flyway/flyway
+    command: -url=jdbc:postgresql://db/postgres -schemas=public -user=postgres -password=password -connectRetries=5 migrate
+    volumes:
+      - ./db-migrations:/flyway/sql
+    depends_on:
+      - db
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_DB: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
```

Comment:
Another todo for later down the line - pass credentials as environment variable to make it more secure